akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/CapturedLogEvent.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.testkit.typed
+
+import java.util.Optional
+
+import akka.actor.typed.LogMarker
+import akka.event.Logging.LogLevel
+import akka.util.OptionVal
+import akka.util.OptionVal.{ None, Some }
```

Comment:
I'd rather use these as `OptionVal.Some` and `OptionVal.None` to avoid confusing them with standard ones